### PR TITLE
Build tests for .NET framework on Linux too; just don't run those tests

### PR DESCRIPTION
**Pull request by jskeet**

### CODE REVIEW

File: tools/Google.Cloud.Tools.ProjectGenerator/Program.cs [hunk 1]
```diff
@@ -82,7 +82,8 @@ public class Program
             { "xunit.runner.visualstudio", "2.4.1" },
             { "Xunit.SkippableFact", "1.3.12" },
             { "Moq", "4.12.0" },
-            { "System.Linq.Async", "4.0.0" }
+            { "System.Linq.Async", "4.0.0" },
+            { ReferenceAssembliesPackage, ReferenceAssembliesVersion }
```
Comment (chrisdunelm) [inline tools/Google.Cloud.Tools.ProjectGenerator/Program.cs]: Am I missing something... `ReferenceAssembliesPackage` and `ReferenceAssembliesVersion` don't seem to be defined?

Comment (jskeet) [reply tools/Google.Cloud.Tools.ProjectGenerator/Program.cs]: They're constants that were already defined elsewhere.

**Comment by jskeet** [comment]
Will have a look at why the Spanner snippets are failing to build.

**Comment by jskeet** [comment]
Okay, I know what was wrong - I've pushed an extra commit to fix a couple of projects. I'll run a complete build on another machine.

**Pull request merged**